pages/test_analytics/flaky_test_assignment.md

```diff
@@ -0,0 +1,32 @@
+# Flaky test assignment
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign flaky tests to [Teams](/docs/team_management/permissions).
+
+## Enabling flaky test assignments
+
+To enable assignments you must have at least one team that has access to your suite. You may need to ask your admin to enable the teams feature, and then create teams via the organization settings page.
+
+<%= image "flaky-test-no-teams.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing dropdown menu prompting user to create teams" %>
+<%= image "team-settings.png", width: 1960/2, height: 630/2, alt: "Team page for assigning test suites" %>
+
+## Assigning a test
+
+When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the assignment dropdown. From here you may assign, reassign or remove the assignment.
+
+>🚧 Assignment permissions
+> All team members have the ability to create, update or remove an assignment, this feature is not restricted to admins.
+
+Tests that are assigned to a team will update to display a badge indicating as such. This badge is also visible on issues in the run page.
```

Comment:
```suggestion
Tests that are assigned to a team will be updated to display a badge indicating as such. This badge is also visible on issues in the run page.
```

Reason: The verb 'update' is transitive and requires an object when used in a sentence/clause.